src/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -104,6 +104,34 @@
     <Compile Include="Binary\BinaryWriterUnitTests.cs" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="..\Common\src\System\MutableDecimal.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="ParsersAndFormatters\Formatter\FormatterTestData.cs" />
```

Comment:
Since we will likely have Encoders, it might make more sense to just have the following directories at the root, rather than nested within "ParsersAndFormatters":
- Formatter
- Parser
- Encoder

Reply:
I'm not sure - the Formatter and Parser test-bed has a lot of shared infrastructure that wouldn't be useful for Encoders.